src/pages/home/report/ReportActionCompose.js

```diff
@@ -333,12 +417,14 @@ class ReportActionCompose extends React.Component {
      * @param {String} emoji
      */
     addEmojiToTextBox(emoji) {
+        const emojiWithSpace = `${emoji} `;
         const newComment = this.comment.slice(0, this.state.selection.start)
-            + emoji + this.comment.slice(this.state.selection.end, this.comment.length);
+      + emojiWithSpace
+      + this.comment.slice(this.state.selection.end, this.comment.length);
         this.setState(prevState => ({
             selection: {
-                start: prevState.selection.start + emoji.length,
-                end: prevState.selection.start + emoji.length,
+                start: prevState.selection.start + emojiWithSpace.length,
```

Comment:
Why is the start and end the same here?

Reply:
If you want to paste an emoji in a specific position, both the start and end positions should be the same. Incrementing the end position will capture the character following the emoji.